openedx/core/djangoapps/user_authn/views/password_reset.py

```diff
@@ -747,6 +749,7 @@ def post(self, request, **kwargs):
             if form.is_valid():
                 form.save()
                 reset_status = True
+                update_session_auth_hash(request, user)
```

Comment:
Can we move this just before the email sending call at L775/L778 to make sure the password reset was a success before updating the hash?

Reply:
Sure, 👍